Sources/SKTestSupport/SkipUnless.swift

```diff
@@ -372,8 +372,8 @@ package actor SkipUnless {
       let output = try result.utf8Output()
       switch output {
       case "": return false
-      case "let x = 1": return true
-      default: throw GenericError("Received unexpected formatting output: \(output)")
+      case "let x = 1\n": return true
```

Comment:
Could just strip `output`?

Reply:
🤷🏽 This check will disappear pretty quickly anyway once we no longer support building and testing SourceKit-LSP using a Swift 6.0 compiler.